packages/block-library/src/navigation/index.php

```diff
@@ -67,6 +67,17 @@ function build_css_font_sizes( $attributes ) {
 	return $font_sizes;
 }
 
+/**
+ * Filters out links with no labels
+ *
+ * @param array $block Block to check label attribute on.
+ *
+ * @return boolean
+ */
+function has_navigation_label( $block ) {
```

Comment:
I realize the other functions in this file are not namespaced either, but let's lead by good example until #18589 is in.
Maybe add a leading `gutenberg_`?